src/lib.rs

```diff
@@ -159,6 +159,7 @@ pub struct PrinterBuilder<'a, Writer: Write> {
     panels: u64,
     group_size: u8,
     base: Base,
+    little_endian_dump: bool,
```

Comment:
I would prefer if we add a `enum Endianness { Little, Big }` instead of using a bool internally.